internal/security/bootstrapper/config/types.go

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright 2021 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+
+package config
+
+// BootStrapperInfo defines the first stage gate info
+// It is the first stage gate of security bootstrapping
+type BootStrapperInfo struct {
+	Host      string
+	StartPort int
+}
+
+// ReadyInfo defines the ready stage gate info
+// It is the last stage gate of security bootstrapping for
+// Kong, and all other Edgex core services
+type ReadyInfo struct {
+	ToRunPort int
+}
+
+// TokensInfo defines the tokens ready stage gate info
+// for the secretstore (aka. vault-worker)
+type TokensInfo struct {
+	ReadyPort int
+}
+
+// VaultWorkerInfo defines the fields related to
+// stage gating the secretstore, vault-worker
+type VaultWorkerInfo struct {
+	Host   string
+	Tokens TokensInfo
+}
+
+// RedisInfo defines the fields related to
+// stage gating the redis db
+type RedisInfo struct {
+	Host      string
+	Port      int
+	ReadyPort int
+}
+
+// ConsulInfo defines the fields related to
+// stage gating the Consul registry
+type ConsulInfo struct {
```

Comment:
IMO, `Consul` => `Registry` and since we are using `REGISTRY_HOST` & `REGISTRY_PORT` override envs, we don't need `Host` & `Port` here.

Reply:
ok, in golang portion of code I do want to keep it all coming from configurations so that there is no segmented definition of structure in terms of registry. I would prefer the readyport is also part of RegistryInfo as well. 
And I thought we had agreement about the naming as I asked you in the slack.

Reply:
Sorry, I concentrated on the structure, not the names. RegistryInfo is part of the services config from go-mod-bootstrap. Here you are just adding the readyport for stage gate needs.